bb-new.html

```diff
@@ -0,0 +1,282 @@
+          <button syle="display: inline-block">BLACKBERRY QNX</button>
+        </div>
+      </div>                                 
+      <div class="row">
+        <div class="column" style="background-color: navy">
+          <h2>SPOTLIGHT ON INNOVATION</h2>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column" style="background-color: navy">
+          <a href="#report">REPORT</a>
+          <p>The BlackBerry 2021 Threat Report Examines and Analyzes the Major Cybersecurity Threats of 2020</p>
+          <button>LEARN MORE</button>
+        </div>
+        <div class="column" style="background-color: navy">
+          <a href="#report">REPORT</a>
+          <p>BlackBerry is Recognized as a Leader in IDC MarketScape for Worldwide Unified Endpoint Management</p>
+          <button>GET REPORT</button>
+        </div>
+        <div class="column" style="background-color: navy">
+          <a href="#report">REPORT</a>
+          <p>Gartner Report: Innovation Insight for Unified Endpoint Security</p>
+          <button>GET REPORT</button>
+        </div>
+        <div class="column" style="background-color: navy">
+          <a href="#report">REPORT</a>
+          <p>BlackBerry Undercovers Massive Hack-For-Hire Group Targeting Governments, Businesses, Human Rights Groups and Influential Individuals</p>
+          <button>GET REPORT</button>
+        </div>
+      </div>              
+      <div class="row">
+        <div class="col-2">
+          <h3>LATEST NEWS</h3>
+        </div>       
+        <div class="col-2">
+          <h4>CONTACT SALES</h4>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column">
+          <div class="footer">
+            <div class="row">
+              <div class="column">
+                <ul>
+                  <li class="footer-list-item"><a href="#facebook">Facebook icon</a></li>
+                  <li class="footer-list-item"><a href="#twitter">Twitter icon</a></li>
+                  <li class="footer-list-item"><a href="#linkdeln">Linkdln icon</a></li>
+                  <li class="footer-list-item"><a href="#youtube">Youtube icon</a></li>
+                  <li class="footer-list-item"><a href="#instagram">Instagram icon</a></li>
+                </ul>
+              </div>
+              <div class="column">
+                <a href="#contact">CONTACT US</a>
+              </div>
+              <div class="column">
+                <a href="#support">SUPPORT</a>
+              </div>
+              <div class="column">
+                <a href="#careers">CAREERS</a>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-5">
```

Comment:
Please look at attached image below (it's an image of the code, so you can't copy it line by line):

The footer should be adjusted to that structure.

![image](https://user-images.githubusercontent.com/4802600/112346034-f5683980-8c9b-11eb-8087-0ecaa6c31597.png)

Reply:
This code is not updated with what I have currently, this is why yesterday I asked you to ignore this PR. Why is there a nav tag?

Reply:
<div class="row">
        <div class="column">
          <div class="footer">
            <div class="row">
              <div class="column">
                <ul>
                  <li class="footer-list-item"><a href="#facebook">Facebook icon</a></li>
                  <li class="footer-list-item"><a href="#twitter">Twitter icon</a></li>
                  <li class="footer-list-item"><a href="#linkdeln">Linkdln icon</a></li>
                  <li class="footer-list-item"><a href="#youtube">Youtube icon</a></li>
                  <li class="footer-list-item"><a href="#instagram">Instagram icon</a></li>
                </ul>
              </div>
              <div class="column">
                <a href="#contact">CONTACT US</a>
              </div>
              <div class="column">
                <a href="#support">SUPPORT</a>
              </div>
              <div class="column">
                <a href="#careers">CAREERS</a>
              </div>
            </div>
            <div class="row">
              <div class="col-5">
                <h3>Corporate</h3>
                <ul>
                  <li class="footer-list-item"><a href="#company">Company</a></li>
                  <li class="footer-list-item"><a href="#newsroom">Newsroom</a></li>
                  <li class="footer-list-item"><a href="#investors">Investors</a></li>
                  <li class="footer-list-item"><a href="#event">Event & Webinars</a></li>
                  <li class="footer-list-item"><a href="#career">Careers</a></li>
                  <li class="footer-list-item"><a href="#leadership">Leadership</a></li>
                  <li class="footer-list-item"><a href="#responsibility">Corporate Responsibility</a></li>
                  <li class="footer-list-item"><a href="#certifications">Certifications</a></li>
                  <li class="footer-list-item"><a href="#customer">Customer Sucess</a></li>
                </ul>
              </div>
              <div class="col-5">
                <h3>Developers</h3>
                <ul>
                  <li class="footer-list-item"><a href="#enterprise">Enterprise Platform & Apps</a></li>
                  <li class="footer-list-item"><a href="#qnx">BlackBerry QNX Developer</a></li>
                  <li class="footer-list-item"><a href="#network">Network</a></li>
                </ul>
              </div>
              <div class="col-5">
                <h3>Blog</h3>
                <ul>
                  <li class="footer-list-item"><a href="#threat">BlackBerry Threat Vector Blog</a></li>
                  <li class="footer-list-item"><a href="#developer">Developers Blog</a></li>
                  <li class="footer-list-item"><a href="#help">Help Blog</a></li>
                </ul>
              </div>
              <div class="col-5">
                <h3>Legal</h3>
                <ul>
                  <li class="footer-list-item"><a href="#overview">Overview</a></li>
                  <li class="footer-list-item"><a href="#accessibility">Accessibility</a></li>
                  <li class="footer-list-item"><a href="#patents">Patents</a></li>
                  <li class="footer-list-item"><a href="#trademark">Trademark</a></li>
                  <li class="footer-list-item"><a href="#privacy">Privacy Policy</a></li>
                  <li class="footer-list-item"><a href="#uk">UK Modern Slavery Act</a></li>
                </ul>
              </div>
              <div class="col-5">
                <h3>Language</h3>
                <form action="/action_page.php">
                  <select>
                    <label for="language">Language</label>
                    <select name="language" id="language">
                    <option value="english">English</option>
                    <option value="Bahasa Indonesia">Bahasa Indonesia</option>
                    <option value="Deutsch">Deutsch</option>                            
                  </select>                    
                </form>                   
              </div>
          </div>

Reply:
nav tag goes wherever you have navigational links, and does not matter if it's in the header or footer